parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -88,6 +88,7 @@ GPU_NEEDS_FABRIC_MANAGER="{{GPUNeedsFabricManager}}"
 NEEDS_DOCKER_LOGIN="{{and IsDockerContainerRuntime HasPrivateAzureRegistryServer}}"
 IPV6_DUAL_STACK_ENABLED="{{IsIPv6DualStackFeatureEnabled}}"
 OUTBOUND_COMMAND="{{GetOutboundCommand}}"
+BLOCK_OUTBOUND_NETWORK="{{BlockOutboundNetwork}}"
```

Comment:
not critical yet since we aren't actually using it, though we will want to let the karpenter folks know this is a new parameter they'll also need to set on their side